src/Workspaces/Core/Desktop/Workspace/MSBuild/ProjectFile/ProjectFileLoader.cs

```diff
@@ -59,17 +59,20 @@ public LoadedProjectInfo(MSB.Evaluation.Project project, string errorMessage)
 
             try
             {
-                var xmlReader = XmlReader.Create(await ReadFileAsync(path, cancellationToken).ConfigureAwait(false), s_xmlSettings);
-                var collection = new MSB.Evaluation.ProjectCollection();
-                var xml = MSB.Construction.ProjectRootElement.Create(xmlReader, collection);
+                using (var memoryStream = await ReadFileAsync(path, cancellationToken).ConfigureAwait(false))
+                using (var xmlReader = XmlReader.Create(memoryStream, s_xmlSettings))
```

Comment:
❕ Unless you have `s_xmlSettings.CloseInput` set to false, this just became a multiple-dispose situation.

Reply:
Ah, thanks for catching - I will step back and just add a suppression with a tracking issue.